tsconfig.json

```diff
@@ -18,6 +18,7 @@
       "@wagmi/chains": ["packages/core/node_modules/@wagmi/chains/dist"],
       "@wagmi/core": ["packages/core/src"],
       "@wagmi/core/*": ["packages/core/src/*"],
+      "abitype": ["packages/core/node_modules/abitype/dist"],
```

Comment:
Should look [into this](https://github.com/microsoft/TypeScript/issues?q=is%3Aissue+is%3Aopen+TS2742) at some point 😃

Reply:
😅